lib/assets/javascripts/markdown-it-diaspora-linkify.js

```diff
@@ -2,69 +2,11 @@
-},{}],4:[function(require,module,exports){
 // Replace link-like texts with link nodes.
 //
 'use strict';
 
-var urlRegex = require('url-regex');
-var LINK_SCAN_RE = /www|@|\:\/\//;
+var urlRegex = /(?:^|\s)(?:(?:https?|ftp):\/\/|mailto:|xmpp:|www[^.\s]*\.)\S+/gi;
```

Comment:
That doesn't support `mailto:`, `xmpp:` and `wwwb` etc. It also doesn't support parenthesis inside of URLs.